scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -994,7 +994,7 @@ def _create_build_dir_link(self, links_dir_path, instance):
         link_path = os.path.join(links_dir_path, link_name)
 
         if os.name == "nt":  # if OS is Windows
-            command = ["mklink", "/J", f"{link_path}", f"{instance.build_dir}"]
+            command = ["mklink", "/J", f"{link_path}", instance.build_dir.replace("/", "\\")]
```

Comment:
do we know where in the code the forward slash is being set? I would fix it there instread of string manipulation.

Reply:
The forward slash is set in this line:
https://github.com/zephyrproject-rtos/zephyr/blob/9948c29885860b9800257f5a1fd466eb046b426e/scripts/pylib/twister/twisterlib/testsuite.py#L453C9-L453C102
If the replace is removed from there, mklink works fine. All twister tests passed on Linux. Only concern is, that testsuite name on Windows is different (different slashes) after the change. Can it have any impact?

Reply:
dbed2512490687083aca4b97d159763a6c8e0004

Reply:
IMO better to keep same testsuite name between different systems. Here, to get windows path, you can try os.path.normpath instead of replace.

Reply:
Used os.path.normpath instead of replace.